internal/k8s/crdcontroller.go

```diff
@@ -69,6 +69,22 @@ func NewIstioCRDInformers(cs istiocrd.Interface) {
 	})
 }
 
+func isHostRuleUpdated(oldHostRule, newHostRule *akov1alpha1.HostRule) bool {
+	if oldHostRule.ResourceVersion == newHostRule.ResourceVersion {
```

Comment:
@swathinsankaran: I forgot once scenario. I think check should be `if resource version are not same then return true`. Reason: With current logic, `status` code will not be hit until unless there is update to CRD object. As per customer logs, there were no updates to hostrule after timing issue happened. So in that scenario AKO will never do update to VS.

Reply:
We are doing extra handling when the AKO updates the status which should resolve this issue. Periodic processing of the CR is not required as we will end up in unnecessary checksum calculations every 30 seconds.